pkg/volume/csi/csi_plugin.go

```diff
@@ -186,29 +185,29 @@ func (p *csiPlugin) Init(host volume.VolumeHost) error {
 
 	csiClient := host.GetKubeClient()
 	if csiClient == nil {
-		klog.Warning(log("kubeclient not set, assuming standalone kubelet"))
+		klog.InfoS(log("kubeclient not set, assuming standalone kubelet"))
```

Comment:
Is there a "warning" for structured logging?

Reply:
There is no WarningS, so reduce the log level